burstphoto/texture/texture.swift

```diff
@@ -78,10 +79,16 @@ func add_texture_highlights(_ in_texture: MTLTexture, _ out_texture: MTLTexture,
 }
 
 
-func add_texture_exposure(_ in_texture: MTLTexture, _ out_texture: MTLTexture, _ norm_texture: MTLTexture, _ exposure_bias: Int, _ white_level: Int, _ black_level: [Int]) {
+func add_texture_exposure(_ in_texture: MTLTexture, _ out_texture: MTLTexture, _ norm_texture: MTLTexture, _ exposure_bias: Int, _ white_level: Int, _ black_level: [Int], _ color_factors: [Double]) {
     
     let black_level_mean = 0.25*Double(black_level[0] + black_level[1] + black_level[2] + black_level[3])
     
+    let color_factor_mean = 0.25*(color_factors[0]+2.0*color_factors[1]+color_factors[2])
```

Comment:
I need to come back to this and fix it in #44 after this PR is merged in.

Reply:
I am unsure how I can resolve this comment.

Reply:
This one you can close. It was for me so that I can see it in my PR since I need to handle it in that PR.